setup.cfg

```diff
@@ -62,3 +62,7 @@ convention = pep257
 # D202: Ignore a blank line after docstring (collision with Python Black in decorators)
 add-ignore = D104,D107,D202
 max-line-length = 120
+
+[isort]
+known_third_party = torch,pytest
```

Comment:
what is the diff to requirements.txt?

Reply:
What do you mean? I did not change requirements.txt.

Reply:
maybe I am missing something, but why do we need to list some libs here?
all third-party libs are already listed in requirements.txt

Reply:
This tells isort that torch and pytest should be treated as third party imports. isort then separates them from the other imports by an empty newline:

https://github.com/timothycrosley/isort#how-does-isort-work

Reply:
cc: @PyTorchLightning/core-contributors

Reply:
Can we somehow parse all requirements as third party and add them here automatically? Otherwise this is just an additional palce we need to keep up to date.